Automattic/sensei-post-to-course
# Remove coenjacobs/mozart dependency #4

Fixes #4 

### Changes proposed in this Pull Request

* Remove unused dependency `coenjacobs/mozart`.

### Testing instructions


* Install plugin.
* Make sure you have some posts.
* Go to Tools -> Post to Course Creator.
* Create a course from posts
* Make sure course and it's lessons were created for all the posts in the category that you selected.


### New/Updated Hooks

* n/a


### Deprecated Code

* n/a


### Screenshot / Video
* n/a
